source/renderer/app/stores/HardwareWalletsStore.ts

```diff
@@ -2293,26 +2293,18 @@ export default class HardwareWalletsStore extends Store {
     }
 
     const fee = formattedAmountToLovelace(flatFee.toString());
-
     const ttl = this._getTtl();
 
-    const absoluteSlotNumber = this._getAbsoluteSlotNumber();
-
     try {
       const signedTransaction = await signTransactionTrezorChannel.request({
-        // @ts-ignore ts-migrate(2322) FIXME: Type '{ path: string; prev_hash: string; prev_inde... Remove this comment to see the full error message
         inputs: inputsData,
         outputs: outputsData,
         fee: fee.toString(),
         ttl: ttl.toString(),
-        validityIntervalStartStr: absoluteSlotNumber.toString(),
```

Comment:
Dead code - This parameter is a copy-paste from Ledger implementation and wasn't actually consumed anywhere.